app/components/providers/open-request/template.hbs

```diff
@@ -49,8 +49,9 @@
   </div>
   <div class='d-grid gap-2'>
     <Shared::Button
-      class='btn-success btn-lg mt-5'
+      class='btn-success mt-5'
       @label='Create request'
+      @size='btn-lg'
```

Comment:
How about:
```suggestion
      @size='lg'
```

and make getter inside shared component something like and concat to basic class:
```js
get size() {
  const { size } = this.args
  const isCorrectSize = ['sm', 'lg'].includes(size);
  return isCorrectSize ? `btn-${size}` : '';
}
```